src/js/components/addData/AddDataMeta.jsx

```diff
@@ -220,6 +255,7 @@ export default class AddDataMeta extends React.Component {
                                 </div>
                             </div>
                         </div>
+                        <Modal onClose={this.closeModal} isOpen={this.state.showModal} buttons={button} content={this.state.modalMessage}/>  
```

Comment:
Why are you passing the close button in as a prop when you can just have the button be inside the modal object?